RInAppMessaging/Classes/IAMPreferenceBuilder.swift

```diff
@@ -27,6 +27,10 @@
 
     @objc @discardableResult
     public func build() -> IAMPreference {
+        if BundleInfo.applicationId?.starts(with: "jp.co.rakuten") == true, Bundle.tests == nil {
+            assert(preference.accessToken == nil || preference.accessToken != nil && preference.userId != nil,
```

Comment:
About `preference.accessToken != nil && preference.userId != nil`, it means that it crashes if accessToken is not nil and userId is not nil, right?
Maybe the condition could be `preference.accessToken != nil && preference.userId == nil`?
So It crashes if the userId is nil even if there is a non nil accessToken.

Reply:
The condition should say that "if you specify accessToken you must also specify userId". The following combinations of (userId, accessToken, result) should apply (nil, value, crash), (nil, nil, ok), (value, nil, ok), (value, value, ok). Assert crashes if condition returns false so you condition seems to do the opposite or maybe I misunderstand something.

Reply:
Ok I understand the condition now. Thanks.